### PR TITLE
feat(base-driver): prefer system unzip

### PR DESCRIPTION
**Pull request by boneskull**
Backport of https://github.com/appium/appium-base-driver/pull/503 to the `2.0` branch of `appium/appium`

### CODE REVIEW

**Review by jlipps — APPROVED** [review]

**Review by mykola-mokhnach — APPROVED** [review]
Please only merge this PR after the corresponding change is merged to appium/support and published there.
I hope that affected module versions are bumped automatically after merge.

**Pull request merged**
